include/file_urlreader.h

```diff
@@ -0,0 +1,24 @@
+#ifndef NEWSBOAT_FILE_URLREADER_H_
+#define NEWSBOAT_FILE_URLREADER_H_
+
+#include <string>
+
+#include "urlreader.h"
+
+namespace newsboat {
+
+class file_urlreader : public urlreader {
```

Comment:
Didn't you want to use UpperCamelCase for classes in new code? Technically speaking a refactoring produces new code. :-P But this class existed before, so the class itself is not new, yeah.

Reply:
I guess the rationale is that in this step we only extract this class from the file. During this step the name should not be changed. The same way you don't change the code during transition -- that would tell why the curly braces are missing for one-liners.

Reply:
Yeah, this was just me moving things around, not even refactoring.

CameCase issue was never resolved, actually. That's a mistake, I'll got fix it; thanks for the nudge! :)